modules/xfeatures2d/test/test_features2d.cpp

```diff
@@ -589,7 +589,7 @@ TEST(Features2d_SIFT_using_mask, regression)
 }
 
 #ifdef OPENCV_ENABLE_NONFREE
-TEST(DISABLED_Features2d_SURF_using_mask, regression)
```

Comment:
I prpose not touch test names to make branches comparable.

Reply:
Ok. I'll revert the them in both `opencv` and `opencv_contrib`.